### ISSUE TITLE
powershell frozen / not responding

### ISSUE THREAD

**Issue by nogetrandom** [Sycnex/Windows10Debloater#382]
Hi!

I'm running the "Remove bloatware with custom blocklist" from the GUI.psi script and after an hour it's still not responding. 
I'm aware that the process can take a while and if that's the reason then never mind this, but if it's not, which I'm starting to suspect, then I'd like to know how long this usually takes. Just so I know if I should abort and retry.

Thanks for hearing me out.

![image](https://user-images.githubusercontent.com/62009756/117163685-2f148000-adc4-11eb-8626-560ff4513f5e.png)
![image](https://user-images.githubusercontent.com/62009756/117163784-4a7f8b00-adc4-11eb-9755-8f66b7b6580e.png)

**Comment by pratyakshm** [comment]
Maybe relaunch the ps1 file and retry again.

**Comment by Sycnex** [comment]
Agreed. There’s no reason it should take that long, maybe 30-60 seconds.

**Comment by nogetrandom** [comment]
thanks guys. I restated it and it happened a few more times, but in the end it got the job done. It seems like it was struggling with Cortana.

**Comment by pratyakshm** [comment]
That's most likely due to the fact that the script doesn't remove standalone Cortana app from 2004 and up, yet.

Sent from ProtonMail mobile

-------- Original Message --------
